src/bindings/js/node/tests/tensor.test.js

```diff
@@ -1,10 +1,27 @@
-var ov = require('../build/Release/ov_node_addon.node');
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+let ov = require('../build/Release/ov_node_addon.node');
```

Comment:
Maybe I ask this question all the time., sorry in advance) Why is let, not const?

Reply:
Sorry 😄 Those parts of code are old (~1 year) and I am mostly focused on new changes